grails-app/conf/application.groovy

```diff
@@ -19,7 +19,7 @@ maxPerPage = 8
 
 totalMax = 100
 
-//https://media-dev.ooica.net/CavaMedia
+jwPlayerKey = "mj9LTtvpJ1Dy6DpIHY30xijBS9IQ4QJdGQoKHaS4D7o="
```

Comment:
Should this be in the code? or it doesn't matter since it's a public key anyways.

Reply:
yeah it's a js variable so it's public